### PR TITLE
Add NeMo Parakeet post

### PR DESCRIPTION
**Pull request by titu1994**
# What does this PR do ?

Adds the content for Parakeet post.

**Collection**: [ASR]

# Changelog 
- Add parakeet post
- Update template to refer to 2024

# Before your PR is "Ready for review"
**Pre checks**:
- [x] Make sure you read and followed [Contributor guidelines](https://github.com/NVIDIA/NeMo/blob/main/CONTRIBUTING.md)
- [ ] Did you write any new necessary tests?
- [ ] Did you add or update any necessary documentation?
- [ ] Does the PR affect components that are optional to install? (Ex: Numba, Pynini, Apex etc)
  - [ ] Reviewer: Does the PR have correct import guards for all optional libraries?
  
**PR Type**:
- [x] New Feature
- [ ] Bugfix
- [ ] Documentation

### CODE REVIEW

File: docs/blogs/posts/2024/2024-01-parakeet.md [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+* [NeMo Parakeet Models on HuggingFace](https://huggingface.co/models?library=nemo&sort=trending&search=parakee)
+* [NVIDIA NeMo Webpage](https://github.com/NVIDIA/NeMo)
+* [NVIDIA NeMo ASR Documentation](https://docs.nvidia.com/deeplearning/nemo/user-guide/index.html)
+* Papers:
+    * [Fast Conformer with Linearly Scalable Attention for Efficient Speech Recognition](https://arxiv.org/abs/2305.05084)
+    * [Investigating End-to-End ASR Architectures for Long Form Audio Transcription](https://arxiv.org/abs/2309.09950)
+
+## About NVIDIA NeMo
+
+[NVIDIA NeMo](https://nvidia.github.io/NeMo/) is an open-source toolkit for building state-of-the-art conversational AI models. It provides a collection of pre-built modules for speech recognition, natural language processing, and text-to-speech, allowing developers to quickly and easily create custom conversational AI applications.
```
Comment (nithinraok) [inline docs/blogs/posts/2024/2024-01-parakeet.md:103]: Lets remove this part for NeMo blog.

File: docs/blogs/posts/2024/2024-01-parakeet.md [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+Once installed, you can evaluate a list of audio files as follows:
+```python
+import nemo.collections.asr as nemo_asr
+asr_model = nemo_asr.models.ASRModel.from_pretrained(model_name="nvidia/parakeet-rnnt-1.1b")
+transcript = asr_model.transcribe(["some_audio_file.wav"])
+```
+
+## Long-form Speech Inference
+
+Once you have a Fast Conformer model loaded, can easily modify the attention type to limited context attention after building the model. Additionally, you can also apply audio chunking for the subsampling module to perform inference on huge audio files!
```
Comment (nithinraok) [inline docs/blogs/posts/2024/2024-01-parakeet.md:76]: Can you add a sentence that published model was trained with full attention and using limited context will affect the accuracy.

**Review by nithinraok — APPROVED** [review]
Thanks

File: docs/blogs/posts/2024/2024-01-parakeet.md [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+* **Open-source and extensibility:** Built on NVIDIA NeMo, allowing for seamless integration and customization.
+* **Pre-trained checkpoints:** Ready-to-use models for inference or fine-tuning.
+* **Permissive license:** Released under CC-BY-4.0 license, model checkpoints can be used in any commercial application.
+
+Parakeet is a major step forward in the evolution of conversational AI. Its exceptional accuracy, coupled with the flexibility and ease of use offered by NeMo, empowers developers to create more natural and intuitive voice-powered applications. The possibilities are endless, from enhancing the accuracy of virtual assistants to enabling seamless real-time communication.
+
+The Parakeet family of models achieves state-of-the-art numbers on the [HuggingFace Leaderboard](https://huggingface.co/spaces/hf-audio/open_asr_leaderboard). Users can try out the [parakeet-rnnt-1.1b](https://huggingface.co/nvidia/parakeet-rnnt-1.1b) firsthand at the [Gradio demo](https://huggingface.co/spaces/nvidia/parakeet-rnnt-1.1b). To access the model locally and explore the toolkit, visit the [NVIDIA NeMo Github page](https://github.com/NVIDIA/NeMo).
```
Comment (Vaibhavs10) [inline docs/blogs/posts/2024/2024-01-parakeet.md:47]: Maybe we can add a screenshot here? IMO people appreciate screenshots quite a lot.

File: docs/blogs/posts/2024/2024-01-parakeet.md [hunk 4]
```diff
@@ -0,0 +1,102 @@
...
+# Enable chunking for subsampling module
+asr_model.change_subsampling_conv_chunking_factor(1)  # 1 = auto select
+
+# Transcribe a huge audio file
+asr_model.transcribe(["<path to a huge audio file>.wav"])  # 10+ hours !
+```
+
+## Additional Resources
+
+* [HuggingFace ASR Leaderboard](https://huggingface.co/spaces/hf-audio/open_asr_leaderboard)
```
Comment (Vaibhavs10) [inline docs/blogs/posts/2024/2024-01-parakeet.md:96]: Might be good to also link the evaluation codebase used by the leaderboard for NeMo somewhere as well?

**Review by Vaibhavs10 — APPROVED** [review]
Love itt! left two minor nits.

**Review by nithinraok — APPROVED** [review]
LGTM

**Pull request merged**